controlplane/kubeadm/controllers/remediation_test.go

```diff
@@ -300,15 +301,15 @@ func TestCanSafelyRemoveEtcdMember(t *testing.T) {
 
 		g.Expect(testEnv.Cleanup(ctx, m1)).To(Succeed())
 	})
-	t.Run("Can't safely remediate 2 machine CP", func(t *testing.T) {
+	t.Run("Can't safely remediate 2 machine CP without additional etcd member failures", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("Can safely remediate 2 machine CP without additional etcd member failures", func(t *testing.T) {
```

Reply:
Fixed, also added a test for "Can't safely remediate 2 machines CP with one additional etcd member failure"